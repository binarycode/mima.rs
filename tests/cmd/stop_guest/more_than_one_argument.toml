args = 'stop-guest one two'
status.code = 2
