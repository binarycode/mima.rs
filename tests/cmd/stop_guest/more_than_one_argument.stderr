error: Found argument 'two' which wasn't expected, or isn't valid in this context

USAGE:
    mima stop-guest [OPTIONS] <GUEST_ID>

For more information try --help
